### PR TITLE
[dv,chip_sw_pwrmgr] Fix all_reset_reqs test

### PR DESCRIPTION
**Pull request by matutem**
The test must fail if reset_info is unexpected.

Signed-off-by: Guillermo Maturana <maturana@google.com>

### CODE REVIEW

**Review by jeoongp — COMMENTED** [review]
@matutem, thanks for correction. 
I have another PR related to this test : https://github.com/lowRISC/opentitan/pull/12757
It has been waiting for merge for one week.. It's blocked by unstable FPGA test glitch which has been recently announced. 
Would you mind waiting for this merge and update on the merged one?

**Comment by matutem** [comment]
@jeoongp No problem waiting. Please let me know when yours is merged.

**Comment by jeoongp** [comment]
> @jeoongp No problem waiting. Please let me know when yours is merged.

After all, it's merged. Thanks for waiting!

**Review by jeoongp — APPROVED** [review]
LGTM

**Pull request merged**
